crawler/src/crawler.ts

```diff
@@ -69,7 +70,10 @@ const processNextBlock = async () => {
     }
 
     // Missing Contracts - Inefficient pattern
-    await parseAndInsertContracts(finalizedHead);
+    if (updateSubContractsCounter > config.subcontractInterval) {
```

Comment:
I don't see where are you incrementing `updateSubContractsCounter` so this condition will never trigger?